Automattic/wp-calypso
# Subscribers Page: tests for SubscriberList and SubscriberRow

## Proposed Changes

This PR adds some test for SubscriberList and SubscriberRow components.

## Testing Instructions

1. Apply this PR, but don't start the application.
2. Execute in the application's root `yarn test-client:watch client/my-sites/subscribers/components/subscriber-list/test/subscriber-row.test.tsx`. The tests should be all green.
3. Execute in the application's root `yarn test-client:watch client/my-sites/subscribers/components/subscriber-list/test/subscriber-list.test.tsx`. The tests should be all green.

## Pre-merge Checklist
